lib/java/ydoc-polyfill/src/main/java/org/enso/ydoc/polyfill/web/Timers.java

```diff
@@ -69,8 +69,10 @@ public Object execute(Value... arguments) {
         yield setInterval(func, normalizedDelay(delay), args);
       }
       case CLEAR_INTERVAL -> {
-        var intervalId = arguments[1].asHostObject();
-        clearInterval(intervalId);
+        if (arguments[1].isHostObject()) {
```

Comment:
Alternative could be
```suggestion
        if (!arguments[1].isNull()) {
```
both variants are likely to pass the new test. As JavaScript `null` as well as `undefined` are `isNull()`.